code/game/objects/mail.dm

```diff
@@ -0,0 +1,230 @@
+
+/obj/item/envelope/engineering
+	icon_state = "mail_eng"
+	possible_contents = list(/obj/item/airlock_electronics,
+	/obj/item/reagent_containers/food/drinks/cans/beer,
+	/obj/item/reagent_containers/food/snacks/candy/confectionery/nougat,
+	/obj/item/mod/module/storage/large_capacity,
+	/obj/item/weldingtool/hugetank,
+	/obj/item/geiger_counter,
+	/obj/item/rcd_ammo,
+	/obj/item/grenade/gas/oxygen,
+	/obj/item/toy/figure/crew/atmos,
+	/obj/item/toy/figure/crew/ce,
+	/obj/item/toy/figure/crew/engineer)
+	job_list = list("Chief Engineer", "Station Engineer", "Engine Technician", "Electrician", "Life Support Specialist", "Atmospheric Technician")
+
+/obj/item/envelope/service
+	icon_state = "mail_serv"
+	possible_contents = list(/obj/item/painter,
+	/obj/item/twohanded/push_broom,
+	/obj/item/gun/energy/floragun,
+	/obj/item/reagent_containers/food/drinks/bottle/fernet,
+	/obj/item/whetstone,
+	/obj/item/reagent_containers/food/drinks/bottle/holywater,
+	/obj/item/stack/ore/tranquillite,
+	/obj/item/stack/ore/bananium,
+	/obj/item/toy/figure/crew/bartender,
+	/obj/item/toy/figure/crew/botanist,
+	/obj/item/toy/figure/crew/chef,
+	/obj/item/toy/figure/crew/clown,
+	/obj/item/toy/figure/crew/hop,
+	/obj/item/toy/figure/crew/chaplain,
+	/obj/item/toy/figure/crew/janitor,
+	/obj/item/toy/figure/crew/librarian,
+	/obj/item/toy/figure/crew/mime)
+	job_list = list("Head of Personnel", "Bartender", "Chef", "Cook", "Culinary Artist", "Butcher", "Botanist", "Hydroponicist", "Botanical Researcher", "Clown", "Mime", "Janitor", "Custodial Technician", "Librarian", "Journalist", "Barber")
+
+/obj/item/envelope/command
+	icon_state = "mail_com"
+	possible_contents = list(/obj/item/flash,
+	/obj/item/storage/fancy/cigarettes/cigpack_robustgold,
+	/obj/item/poster/random_official,
+	/obj/item/book/manual/wiki/sop_command,
+	/obj/item/reagent_containers/food/pill/patch/synthflesh,
+	/obj/item/paper_bin/nanotrasen,
+	/obj/item/reagent_containers/food/snacks/spesslaw,
+	/obj/item/clothing/head/collectable/petehat,
+	/obj/item/toy/figure/crew/captain,
+	/obj/item/toy/figure/crew/lawyer,
+	/obj/item/toy/figure/crew/dsquad)
+	job_list = list("Captain", "Magistrate", "NanoTrasen Representative", "Blueshield", "Internal Affairs Agent", "Human Resources Agent")
+
+/obj/item/envelope/misc
+	possible_contents = list(/obj/item/clothing/under/misc/assistantformal,
+	/obj/item/clothing/under/syndicate/tacticool,
+	/obj/item/clothing/shoes/ducky,
+	/obj/item/toy/plushie/orange_fox/grump,
+	/obj/item/multitool,
+	/obj/item/instrument/piano_synth,
+	/obj/item/toy/crayon/spraycan,
+	/obj/item/clothing/head/cakehat,
+	/obj/item/toy/figure/crew/assistant,
+	/obj/item/toy/figure/owl,
+	/obj/item/toy/figure/griffin)
+	job_list = list("Assistant", "Explorer")
+
+/obj/item/mail_scanner
+	name = "mail scanner"
+	desc = "Confirms deliveries of mail with clients."
+	icon = 'icons/obj/surgery.dmi'
```

Comment:
Currently the autopsy scanner is the sprite for the mail scanner, mostly because I'm currently struggling to get a unique sprite for it.

On the bright side it looks very nice with a loader modsuit. I'll try seeing if I can get a sprite for the mail scanner in the meantime.

Reply:
Could try recoloring the autopsy scanner sprite if you think it fits, wouldnt be hard.

Reply:
Spoke with Christa, gonna wait to see what they come up with for the mail scanner. Worst case scenario I'll an EFPTOS scanner most likely. The autopsy scanner is very bulky.